pkg/server/tests/tls/tls_test.go

```diff
@@ -61,7 +61,7 @@ func newTLSHttpClient(t *testing.T, caFile, certFile, keyFile string) *http.Clie
 // If parentCert and parentCertKey is specified, the new certificate will be signed by the parentCert.
 // Otherwise, the new certificate will be self-signed and is a CA.
 func generateCert(sn int, commonName string, parentCert *x509.Certificate, parentCertKey *rsa.PrivateKey, outKeyFile string, outCertFile string, opts ...func(c *x509.Certificate)) (*x509.Certificate, *rsa.PrivateKey, error) {
-	privateKey, err := rsa.GenerateKey(rand.Reader, 528)
+	privateKey, err := rsa.GenerateKey(rand.Reader, 1024)
```

Comment:
why change this

Reply:
https://pkg.go.dev/crypto/rsa@go1.25.3#hdr-Minimum_key_size
- [GenerateKey](https://pkg.go.dev/crypto/rsa#GenerateKey) returns an error if a key of less than 1024 bits is requested, and all Sign, Verify, Encrypt, and Decrypt methods return an error if used with a key smaller than 1024 bits. Such keys are insecure and should not be used.

Reply:
got